tests/Unit/Evolution/Systems/Cce/CceComputationTestHelpers.hpp

```diff
@@ -0,0 +1,59 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include "DataStructures/ComplexDataVector.hpp"
+#include "DataStructures/ComplexModalVector.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tags.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Systems/Cce/Tags.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "NumericalAlgorithms/Spectral/SwshCoefficients.hpp"
+#include "NumericalAlgorithms/Spectral/SwshFiltering.hpp"
+#include "NumericalAlgorithms/Spectral/SwshTags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/VectorAlgebra.hpp"
+#include "tests/Utilities/MakeWithRandomValues.hpp"
+
+namespace Cce {
+namespace TestHelpers {
+
+// For representing a primitive series of powers in inverse r for diagnostic
+// computations
+template <typename T>
+struct RadialPolyCoefficientsFor : db::SimpleTag {
+  // for the test, we just store a quadratic in radius
+  using type = Scalar<ComplexModalVector>;
+  using tag = T;
+  static std::string name() {
+    return "RadialPolyCoefficientsFor(" + T::name() + ")";
```

Comment:
you can remove this `name()` function since it has a default implementation now.